client/store/current-user/resolvers.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import apiRequest from '@wordpress/api-request';
+
+export default {
+	async getCurrentUser() {
+		const path = '/wp/v2/users/me?context=edit';
```

Comment:
For some reason this is 404'ing for me:

![image](https://user-images.githubusercontent.com/22080/43600311-249f18a0-963f-11e8-90d4-59768b6e7a78.png)